src/writeData/subscriptions/context/subscription.list.tsx

```diff
@@ -30,19 +37,134 @@ export const DEFAULT_CONTEXT: SubscriptionListContextType = {
   loading: RemoteDataState.NotStarted,
   change: (_id: string) => {},
   currentID: null,
+  bulletins: {},
 } as SubscriptionListContextType
 
 export const SubscriptionListContext = React.createContext<
   SubscriptionListContextType
 >(DEFAULT_CONTEXT)
 
+export interface Bulletin {
+  timestamp: number
+  message: string
+}
+
+interface BulletinsMap {
+  [key: string]: Bulletin[]
+}
+
+const sanitizeBulletin = (bulletin: string) => {
```

Comment:
I'd like to pull these bulletin functions / date functions into utils rather than in the list context